CactusTutorial.ipynb

```diff
@@ -368,15 +366,15 @@
     "\n",
     "HydroBase::evolution_method      = \"GRHydro\"\n",
     "\n",
-    "GRHydro::riemann_solver            = \"HLLE\"\n",
-    "GRHydro::GRHydro_eos_type          = \"General\"\n",
-    "GRHydro::GRHydro_eos_table         = \"Ideal_Fluid\"\n",
-    "GRHydro::recon_method              = \"ppm\"\n",
-    "GRHydro::GRHydro_stencil            = 3\n",
-    "GRHydro::bound                     = \"none\"\n",
-    "GRHydro::rho_abs_min               = 1.e-10\n",
-    "GRHydro::GRHydro_atmo_tolerance    = 1.e-3\n",
-    "GRHydro::sources_spatial_order     = 4\n",
+    "GRHydro::riemann_solver         = \"Marquina\"\n",
+    "GRHydro::GRHydro_eos_type       = \"Polytype\"\n",
+    "GRHydro::GRHydro_eos_table      = \"2D_Polytrope\"\n",
+    "GRHydro::recon_method           = \"ppm\"\n",
+    "GRHydro::GRHydro_stencil        = 3\n",
+    "GRHydro::bound                  = \"none\"\n",
+    "GRHydro::rho_abs_min            = 1.e-10\n",
+    "#new finite difference order parameter\n",
```

Comment:
would suggest against using the word "new" here since "new" will turn
into "old" with time. Instead "parameter controlling finite difference
order for Christoffel symbols in GRHydro" which is what it controls
(though I would not be surprised if some of the vector potential code
uses it as well).

Reply:
Agreed

Reply:
done